extra-resources/mods/Mythos/greek.cardset

```diff
@@ -461,7 +573,8 @@ card('Mount Olympus') {
     }
     */
 }
-
+// https://en.wikipedia.org/wiki/Tartarus
+// NO IMAGE FOUND
```

Comment:
Yeah, I added that when I literally could not find a suitable image. Those will display the default image (I'm going to have mod-specific logos soon enough for the mods so it won't look so much out of place with the regular Cardshifter logo)